career/working-and-collaborating-remotely.md

```diff
@@ -0,0 +1,172 @@
+
+### Materials
+- Read [2019 Survey of remote workers](https://buffer.com/state-of-remote-work-2019)
+- Watch [10 tips for productive remote work](https://www.youtube.com/watch?v=0JbeJgcRPMM)
+
+### Lesson
+**Remote work** is defined as working offsite from a traditional office. **Remote collaboration** is defined as working with at least one person who is physically at another site, independent of whether the sites are within a traditional office setting. Both present both pros and cons. For example, some pros would be that remote collaboration saves both the employees and company the costs of travel time, and remote work saves the employee the costs of commuting and the company the costs of office space. On the con side, both remote collaboration and remote work lose the benefits of interpersonal communication that can take place when everyone involved are within proximity. Challenges on the con side are exemplified in the first minute and a half of this video on the [remote work revolution](https://www.youtube.com/watch?v=wUTd35A7I6c), where the speaker notes the following quote:
+
+> 1995: The information superhighway will mean anyone can do anything from anywhere
+>
+> 2015: must be willing to relocate to San Francisco
+
+The quote references a job posting by a tech company that produces software for remote collaboration! It appears they prefer their workers onsite! Because many companies are testing the value of remote work and there is a lack of visual interactions between coworkers, one must be cognizant of how to navigate remote work and collaboration professionally and how to optimize the use of the tools available for professionalism and professional growth.
+
+#### Working from home or a rented office
+- **Work/Life balance** - As shown in the "2019 Survey of remote workers" blog post, one of the main struggles for remote workers is disconnecting from work. When work and life begin to mix together it is very important to have:
+  - A dedicated area(s) for where you do your work
+  - A set of rituals that are done before and after work - such as getting dressed as if going into the office and walking around the block for a "commute", before sitting down at the work station
+  - A separate computer or account to separate work from personal computer time
+  - A discussion about expectations with family or roommates regarding your work time, and a plan as to when and how to interact with others that may share the same household
+- **Time management and productivity** - Another struggle closely related to work/life balance is time management. This  problem arises because as opposed to being around others in a traditional office, there are no social queues for break and lunch times and an environment where you see everyone quiet and working. Tips:
+  - Have a clean and uncluttered work area
+  - Set a specific weekly and daily schedule for work time
+  - Plan your day with a task list
+  - Use tools like Pomodoros and timers
```

Comment:
Can you link to some faves?

For Pomodoro, I've liked https://apps.apple.com/us/app/be-focused-focus-timer/id973134470?mt=12

Reply:
Sure, I put in AS Timer for devises, and marinartimer which is shared over the web